bootstrap_salt/fab_tasks.py

```diff
@@ -140,7 +140,7 @@ def install_minions():
         put_util_script()
         run('wget https://raw.githubusercontent.com/saltstack/salt-bootstrap/%s/bootstrap-salt.sh -O /tmp/bootstrap-salt.sh' % sha)
         sudo('chmod 755 /tmp/bootstrap-salt.sh')
-        sudo('/tmp/bootstrap-salt.sh -A ' + master_prv_ip + ' -p python-boto git v2014.1.4')
+        sudo('/tmp/bootstrap-salt.sh -A ' + master_prv_ip + ' -p python-boto git v2014.7.4')
```

Comment:
We should change this to a parameter to the `setup()` method (with 2014.7.5 as a default) that we pass into install_master and install_minion